### ISSUE TITLE
Staging Review finding: Focus is lost after activating Edit on the Review and Submit page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#117629]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits
**Product:** 21-526ez Disability Benefits Application
**Feature:** Update Form 4142 within 526 to the latest version

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

On the Review & Submit page, for the Edit buttons next to the first section "Supporting Evidence types" and the second section "Non-Va treatment" records, when activating "Edit" the focus is not managed and can be lost. If the user activates TAB after Edit, the screen reader/browser may guess correctly where focus should go. However, if the user (using assistive technology), activates a shortcut or uses arrow navigation, their focus is reset to the top of the page. This can be disorienting for users of assistive technology and disruptive to their completion of the form. 

<details><summary>Link, screenshot or steps to recreate</summary>

To be added

</details>

## Recommended action

Ensure focus is managed when activating Edit. Put focus into the first focusable, or on the section itself, to ensure the user is not taken out of the flow and is aware of where they are. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 2.4.3 Focus Order - Level A](https://www.w3.org/TR/WCAG22/#focus-order)
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by etanb** [comment]
@jasonday Tabbing through the same component on a different page, I was not able to reproduce this -- I'm using a Mac on the latest Chrome build. I've shared a video, but the focus seems to be set properly:

https://github.com/user-attachments/assets/f3a5b564-8448-4e32-80f1-9cbe5130fa10

**Comment by jasonday** [comment]
Hey @etanb 
Try using Safari, and instead of tabbing try arrow navigation, or a keyboard VO shortcut and see if the behavior is reproducible. These are the steps I took when discovering this issue.

Specifically, in my testing (macOS + Voiceover ), if I activated a screen reader shortcut (i.e. VO + Command + H) the shortcut was starting from the top of the page and finding the first heading, versus going to the next heading after the Edit clicked.

**Comment by etanb** [comment]
@comaurice this bug is still happening, and can be reproduced using the specific steps laid out above.
